### ISSUE TITLE
Falcor open source release imminent ?

### ISSUE THREAD

**Issue by Tlvenn** [michaelbpaulson/Falcor#1]
Hi Michael,

Was wondering if you could us give a little update on where you are regarding the release of Falcor to the open source community. Falcor has been introduced to the community in March during Ng Conf, we are now in July, it starts to feel pretty long on the other side of the table so to speak ;)

I saw that Jafar has just posted introduction videos on Falcor, covering many topics which I hope is a sign that the code will be release soon.

Do I read this well and it's likely that the code will be release within a few weeks timeframe (less than a month) ?

Thanks.

**Comment by nikolay** [comment]
:+1:

**Comment by michaelbpaulson** [comment]
Thanks @Tlvenn for the issue!

I don't believe I can give any confirmed dates, but @jhusain could you comment?

**Comment by jhusain** [comment]
At the moment we are undergoing a pretty big code upheaval. We are doing a big factor of some of our internals, and it will take a while to prove that out. And parallel I am working hard on the documentation which is a gating factor at the moment. There's a lot of documentation to write for a library that is so large in scope. Falcor has a pretty fundamentally different approach the data access, and it's really important that we explain how to use it correctly if we expect our users to be successful.

Long story short they're still a lot to do and I'm not able to confirm any date at the moment.

**Comment by jhusain** [comment]
I should clarify I don't think we are months away from releasing. We are moving as quickly as possible to set up a website and finish out the documentation. We just have some fresh code that we want to really prove out for our own comfort level.

**Comment by fdecampredon** [comment]
Just for info dors falcor observable Will ne based on rxjs or latest es spec ?

**Comment by jhusain** [comment]
The new Observable is not far along enough in the standards process yet. The contract is still subject to change (although I think we are converging on a design). As a result we would likely base the Observable on the RX JS contract.

JH

**Comment by jay-porch** [comment]
@jhusain, it's been a month since you wrote *I don't think we are months away*. Any update?

**Comment by Tlvenn** [comment]
Within 2 weeks with the aim to release next week.

**Comment by Tlvenn** [comment]
> https://www.youtube.com/watch?v=WL54eYbTJUw
